src/d3a/d3a_core/sim_results/area_throughput_stats.py

```diff
@@ -22,66 +22,56 @@ class AreaThroughputStats:
     def __init__(self):
         self.results = {}
         self.results_redis = {}
+        self.exported_energy = {}
+        self.imported_energy = {}
 
-    def update(self, area):
-        self.update_results(area)
+    def update(self, area_dict, core_stats, current_market_time_slot_str):
+        self.update_results(area_dict, core_stats, current_market_time_slot_str)
 
-    @staticmethod
-    def _calc_results(area, baseline_value, energy_profile, direction_key):
-        # as this is mainly a frontend feature,
-        # the numbers are rounded for both local and redis results
-        peak_percentage = \
-            round_floats_for_ui(max(energy_profile.values(), default=0.0) / baseline_value * 100)
-        if direction_key == "import":
-            baseline_peak_energy_kWh = round_floats_for_ui(area.baseline_peak_energy_import_kWh)
-        else:
-            baseline_peak_energy_kWh = round_floats_for_ui(area.baseline_peak_energy_export_kWh)
-        return {
-            "peak_percentage": peak_percentage,
-            "baseline_peak_energy_kWh": baseline_peak_energy_kWh,
-         }
+    def update_results(self, area_dict, core_stats, current_market_time_slot_str):
+        area_throughput = core_stats.get(area_dict['uuid'], {}).get('area_throughput', {})
+        imported_peak = round_floats_for_ui(area_throughput.get('imported_energy_kWh', 0.))
+        exported_peak = round_floats_for_ui(area_throughput.get('exported_energy_kWh', 0.))
+        area_results = {
+            "import": {'peak_energy_kWh': imported_peak},
+            "export": {'peak_energy_kWh': exported_peak},
+        }
```

Comment:
Minor but you could merge these lines.

Reply:
The reason I did it this way is due to one line getting longer than 100 and flake8 started complaining.